viability/src/org/labkey/viability/ViabilityManager.java

```diff
@@ -525,7 +525,7 @@ public static void deleteAll(List<ExpData> datas, Container c)
                 dataIDs.add(data.getRowId());
 
             TableSelector ts = new TableSelector(ViabilitySchema.getTableInfoResults(),
-                    new HashSet<String>(Arrays.asList("RowID", "ObjectID")),
+                    new LinkedHashSet<>(Arrays.asList("RowID", "ObjectID")),

```

Comment:
Any reason you're using LinkedHashSet here and PageFlowUtil.set() for the other set of identical columns in this class?

Reply:
Nope, conservation of keystrokes maybe?